### PR TITLE
chore: Types kit

### PR DESCRIPTION
**Pull request by yagopv**
## What it solves
Resolves https://github.com/safe-global/safe-core-sdk/issues/947

## How this PR fixes it
This PR updates imports, versions and some texts

### CODE REVIEW

File: packages/protocol-kit/tests/unit/eip-712.test.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import { SafeTransactionData, OperationType } from '@safe-global/types-kit'
 import chai from 'chai'
 import {
   EIP712_DOMAIN,
   EIP712_DOMAIN_BEFORE_V130,
   generateTypedData,
   getEip712TxTypes
 } from '@safe-global/protocol-kit/utils'
-import { EIP712TypedData } from '@safe-global/safe-core-sdk-types'
+import { EIP712TypedData } from '@safe-global/types-kit'
```
Comment (dasanra) [inline packages/protocol-kit/tests/unit/eip-712.test.ts:9]: Two imports of same package

File: packages/relay-kit/src/packs/gelato/GelatoRelayPack.test.ts [hunk 2]
```diff
@@ -6,8 +6,8 @@ import Safe, {
   estimateSafeDeploymentGas,
   createERC20TokenTransferTransaction
 } from '@safe-global/protocol-kit'
-import { MetaTransactionData, OperationType } from '@safe-global/safe-core-sdk-types'
-import { SafeTransaction } from '@safe-global/safe-core-sdk-types'
+import { MetaTransactionData, OperationType } from '@safe-global/types-kit'
+import { SafeTransaction } from '@safe-global/types-kit'
```
Comment (dasanra) [inline packages/relay-kit/src/packs/gelato/GelatoRelayPack.test.ts:10]: Two imports of the same package

File: packages/protocol-kit/tests/unit/eip-712.test.ts (hunk 1 above)
Comment (yagopv) [reply packages/protocol-kit/tests/unit/eip-712.test.ts]: fixed

File: packages/relay-kit/src/packs/gelato/GelatoRelayPack.test.ts (hunk 2 above)
Comment (yagopv) [reply packages/relay-kit/src/packs/gelato/GelatoRelayPack.test.ts]: fixed

**Review by dasanra — APPROVED** [review]

**Pull request merged**